### ISSUE TITLE
Allow specifying TypeScript compiler options

### ISSUE THREAD

**Issue by JoshuaKGoldberg** [danvk/literate-ts#30]
From this code snippet:

```asciidoc
[source,ts]
----
1337n; // bigint
----
```

...comes this literate-ts complaint:

```
⠦ 2/15: ch02-the_type_system.asciidoc: 2/18 ch02-the_type_system-40
ch02-the_type_system-40: Unexpected TypeScript error: 0:0-5: BigInt literals are not available when targeting lower than ES2020.
```

I'd like to run literate-ts with, say, `"strict": true` and `"target": "esnext"` Typescript compiler options. Could we have, say, a `literate.config.js(on)` config file?

**Comment by danvk** [comment]
I believe it looks at `tsonfig.json` in the directory you run `literate-ts` from. If it doesn't, then it probably should. Would that meet your needs? We could add a `-p` flag to point to a `tsconfig.json` if it doesn't.

**Comment by danvk** [comment]
Also worth noting: there's a `verifier:tsconfig:setting=value` directive if you need to change settings for specific code samples.

**Comment by JoshuaKGoldberg** [comment]
That does meet my needs, thank you! Feels like this is more of a documentation request, then.

**Comment by danvk** [comment]
Yeah, of course! You can probably guess why I'm working on this project again. I don't remember whether you fully got _Learning TypeScript_ passing literate-ts, but I've got to say that adding a caching layer has made it _way_ nicer to work with. So fast!
